.github/workflows/pages-build-and-deploy.yml

```diff
@@ -0,0 +1,45 @@
+name: pages-build-and-deploy
+
+# Only run this workflow when a diepvries/* tag gets pushed
+on:
+  push:
+    tags:
+      - diepvries/*
```

Comment:
I changed the pages and pypi workflow to only run when a `diepvries/*` tag gets pushed. This way we can merge to master, but only deploy when a release tag gets pushed.